font-codegen/Cargo.toml

```diff
@@ -12,6 +12,7 @@ name = "codegen"
 path = "src/main.rs"
 
 [dependencies]
+font-types = { version = "0.0.3", path = "../font-types" }
```

Comment:
QQ, why do we need a version on a path dependency?

Reply:
all dependencies must be on crates.io, and so must have versions. Including the path dependency is just useful during development, so you can make changes across crates and you'll always be using whatever you have checked out.